.github/workflows/backport.yml

```
@@ -0,0 +1,241 @@
+      - created
+
+jobs:
+  backport:
+    name: Backport
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+      issues: write  # For posting feedback comments
+    if: |
+      github.event.issue.pull_request && 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)